nix/shell.nix

```diff
@@ -0,0 +1,54 @@
+{ pkgs, python3, poetry2nix, pyproject, poetrylock, poetry-groups ? [ ], extraPackages ? [ ], extraPythonPackages ? (ps: [ ]), ... }:
+
+let
+  # When running a nix shell, XDG_DATA_DIRS will be populated so that bash_completion can (lazily) find this completion script
+  evap-managepy-completion = pkgs.runCommand "evap-managepy-completion" { } ''
+    mkdir -p "$out/share/bash-completion/completions"
+    install ${../deployment/manage_autocompletion.sh} "$out/share/bash-completion/completions/manage.py.bash"
+  '';
+
+  clean-setup = pkgs.writeShellScriptBin "clean-setup" ''
+    read -p "Delete node_modules/, data/ and evap/localsettings.py? [y/N] "
+    [[ "$REPLY" =~ ^[Yy]$ ]] || exit 1
+    set -ex
+    rm -rf node_modules/ data/ evap/localsettings.py
```

Comment:
I suppose it is fair that if someone really changes localsettings.py, they will have to delete node_modules and data themselves instead of using the script? Not deleting localsettings here is weird since clean-setup should produce a fresh setup for services-full